public/instruct/OpenOrca-OpenChat.json

```diff
@@ -0,0 +1,13 @@
+{
+    "enabled": true,
```

Comment:
I thought it makes sense - by selecting a preset, you'd enable the mode, saving a click. Because why would you switch presets if you didn't want to use them. But if you really don't want that behavior, I'll remove it.